_templates/about-us.html

```diff
@@ -83,3 +84,16 @@ <h3 id="github">{% translate github %}</h3>
   </div>
   {% endfor %}
 </div>
+{% else %}
+{% comment %}
+<!-- if you've cloned bitcoin.org, feel free to fill in your own About
+Us below. It would be appreciated it if you link back to Bitcoin.org,
+but please make it clear that your site is not affiliated with
+Bitcoin.org. -->
```

Comment:
@harding Just a suggestion, maybe we could also mention "and don't forget to update instructions in the README". I don't know if that's really going to be efficient in practice but who knows.